code/src/cljs/sixsq/nuvla/ui/edges/subs.cljs

```diff
@@ -36,6 +36,12 @@
   (fn [edges-status [_ edge-id]]
     (get-in edges-status [edge-id :nuvlabox-engine-version])))
 
+(reg-sub
+  ::one-edge-with-only-major-version
+  :<- [::edges-status]
+  (fn [edges-status [_ ids]]
+    (js/console.error "edges-status" edges-status)
+    (some nil? (map (fn [id] (get-in edges-status [id :nuvlabox-engine-version])) ids))))
```

Comment:
(some (comp nil? :nuvlabox-engine-version second) edges-status)

Reply:
you will get used to it :)

Reply:
Ah damn, `comp` again 😄

Reply:
Ah wait: I cannot use your suggestion because not all nuvlaedges in the page have an entry in edge-status.
That's why I mapped over ids.

But I should be able to use `comp` anyway